tests/integration/compaction/pom.xml

```diff
@@ -38,10 +38,17 @@
   <name>Apache Pulsar :: Tests :: Integration Tests :: Topic Compaction</name>
 
   <dependencies>
```

Comment:
Do you have a plan to move all these into a single module when the migrations are done?

Reply:
yes next pr